runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -223,6 +227,12 @@ func (p *Parser) parseMetricsView(ctx context.Context, node *Node) error {
 		// validation has been done above, only one of these will be set
 		if tmp.Policy.Include != nil {
 			for _, include := range tmp.Policy.Include {
+				if include.Name == "" || include.Condition == "" {
+					return fmt.Errorf("invalid 'policy': 'include' fields must have a valid name and condition")
+				}
+				if !dimensions[include.Name] {
+					return fmt.Errorf("invalid 'policy': 'include' property [%q] does not exists in dimensions list", include.Name)
+				}
```

Comment:
Validation can't happen here – see note earlier in the function:
```
// NOTE: After calling upsertResource, an error must not be returned. Any validation should be done before calling it.
```

Reply:
yeah